### PR TITLE
DOCSP-27268 Compass 1.35 release notes

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Copy and paste from the 1.35 [whats changed](https://github.com/mongodb-js/compass/releases) section. Removed a few items that are not relevant for the release

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27268/release-notes/

## BUILD

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=63beeecddc742b91f29bc0a3

## JIRA

https://jira.mongodb.org/browse/DOCSP-27268

### CODE REVIEW

**Comment by ianf-mongodb** [comment]
Hi @addaleax  could you please have a look at this when you have a moment?

**Review by addaleax — APPROVED** [review]

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,78 @@ Release Notes
    :depth: 1
    :class: twocols
 
+
+|compass| 1.35.0
+----------------
+
+*Released January 11, 2023*
+
+New Features
```
Comment (mdb-ashley) [inline source/release-notes.txt:19]: [suggestion] Should this be "New Features:"?

File: source/release-notes.txt [hunk 2]
```diff
@@ -10,6 +10,78 @@ Release Notes
...
+- Implement new input docs card design (:issue:`COMPASS-6234`)
+- Update scrollbar styles (:issue:`COMPASS-5597`)
+- Cancellable counts (:issue:`COMPASS-5668`)
+- Update aggregations stage components (:issue:`COMPASS-6234`)
+- Enable pipeline as text feature (:issue:`COMPASS-6299`)
+- Index tab UI improvements (:issue:`COMPASS-6323`), (:issue:`COMPASS-6329`)
+- Add refresh document count in aggregation results (:issue:`COMPASS-6156`)
+- Confirm when deleting pipeline (:issue:`COMPASS-4137`)
+
+Bug Fixes
```
Comment (mdb-ashley) [inline source/release-notes.txt:58]: [suggestion] Should this be "Bug Fixes:"?

**Review by mdb-ashley — CHANGES_REQUESTED** [review]
Can you make these small changes? Thanks, Ian!

File: source/release-notes.txt (hunk 1 above)
Comment (ianf-mongodb) [reply source/release-notes.txt]: Updated.

File: source/release-notes.txt (hunk 2 above)
Comment (ianf-mongodb) [reply source/release-notes.txt]: Updated.

**Review by ianf-mongodb — COMMENTED** [review]
@mdb-ashley updated.

**Review by ianf-mongodb — COMMENTED** [review]
Updated.

**Pull request merged**